tracee-rules/input.go

```diff
@@ -66,8 +67,118 @@ func setupTraceeSource(traceeFilePath string) (chan types.Event, error) {
 				res <- event
 			}
 		}
-		f.Close()
+		opts.inputFile.Close()
 		close(res)
 	}()
 	return res, nil
 }
+
+func setupJSONInputSource(opts *traceeInputOptions) (chan types.Event, error) {
+	res := make(chan types.Event)
+	scanner := bufio.NewScanner(opts.inputFile)
+	go func() {
+		for scanner.Scan() {
```

Comment:
restating what I wrote in a general comment. if we decided to always read until EOT, then this should be the behavior here as well. but since it's about to be removed, we can forgive this one this time